saiph/projection.py

```diff
@@ -249,6 +249,7 @@ def get_variable_correlation(
         df_quali = pd.get_dummies(
             df[model.original_categorical].astype("category"),
             prefix_sep=DUMMIES_SEPARATOR,
+            dtype=np.uint8,
```

Comment:
pandas v1 had default `np.uint8`, pandas v2 has it as `bool`.
The latter might break the missing values handling that add a `_True` and `_False` string suffix.